repo/mapping.go

```diff
@@ -126,13 +149,13 @@ func GetAllRepos() ([]string, error) {
 	return str, nil
 }
 
-func GetRepo() []DeviceMapping {
+func GetRepo() *DeviceCollection {
```

Comment:
__[golint]__
* [golint] exported function GetRepo should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->